libcore/marlin-undostack-manager.c

```diff
@@ -370,18 +370,21 @@ marlin_undo_manager_is_undo_redo (MarlinUndoManager *manager)
  * Redoes the last file operation
 ** ****************************************************************/
 void
-marlin_undo_manager_redo (MarlinUndoManager *self,
-                          GtkWidget *parent_view,
-                          MarlinUndoFinishCallback cb,
-                          gpointer callback_data)
+marlin_undo_manager_redo (MarlinUndoManager   *self,
```

Comment:
Maybe rename `marlin_undo_manager_redo_async` ?

Reply:
It is not mandatory to call it *_async as there is a *_finish anyway so whatever you perfer

Reply:
I guess it only matters where there are two similar functions, one sync and one async like in Gio so just leave it.